addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteForward.js

```diff
@@ -28,6 +28,8 @@ import {
     isVisible,
     isUnbreakable,
     isEmptyBlock,
+    isStartOfBehaviorElement,
```

Comment:
Ping @Zynton.
What is your stance on having hard coded reference of a knowledge feature in the editor ? 
Do we already have those kind of stuff ?

Reply:
I'm not aware of a precedent and I'd also prefer to avoid it since knowledge is not a dependency of the editor. We've had some cases of references to mass_mailing which is unfortunate but I don't think directly in the editor though. It would be better to extend this in knowledge to add the special case.

Reply:
I removed this issue from this pr and I will create another one to solve the behavior issue alone since these issues are purely related to editor and not knowledge